cms/templates/ux/reference/container.html

```diff
@@ -1,706 +0,0 @@
-<%inherit file="../../base.html" />
```

Comment:
Reviewers: I've removed all these old UX reference pages as they are very old and no longer render correctly.